cmd/root.go

```diff
@@ -0,0 +1,127 @@
+package cmd
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	config "github.com/kyallanum/athena/v1.0.0/models/config"
+	library "github.com/kyallanum/athena/v1.0.0/models/library"
+	logs "github.com/kyallanum/athena/v1.0.0/models/logs"
+	"github.com/spf13/cobra"
+)
+
+var configFile, logFile string
+
+var rootCmd = &cobra.Command{
+	Use:           "athena [flags]",
+	Short:         "A text and log file parser to discern important information",
+	SilenceUsage:  true,
+	SilenceErrors: true,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(configFile) == 0 {
+			return fmt.Errorf(`required flag(s) "config" not set`)
+		}
+		if len(logFile) == 0 {
+			return fmt.Errorf(`required flag(s) "log-file" not set`)
+		}
+		return nil
+	},
+}
+
+func Execute(logger *log.Logger) error {
+	if err := rootCmd.Execute(); err != nil {
+		return err
+	}
+	logger.Print("Athena v1.0.0 Starting")
+
+	fmt.Println("Getting Configuration File: ", configFile, "...")
+	configuration, err := config.CreateConfiguration(configFile)
+	if err != nil {
+		return err
+	}
+	fmt.Println("Configuration Loaded")
+
+	fmt.Println("Loading Log File: ", logFile, "... ")
+	logFileContents, err := logs.LoadLogFile(logFile)
+	if err != nil {
+		return err
+	}
+	fmt.Println("Log File Loaded")
+
+	library, err := resolveLogFile(logFileContents, configuration)
+	if err != nil {
+		return err
+	}
+
+	if err = printSummary(library); err != nil {
+		return nil
+	}
+	return nil
```

Comment:
Let's add a function called ``errCheck`` in this file that checks for an error thats passed to it, and if it does, then it panics.  Refer to [L14-L18](https://github.com/kyallanum/athena/pull/9/files#diff-2873f79a86c0d8b3335cd7731b0ecf7dd4301eb19a82ef7a1cba7589b5252261L14-L18) of my original main.go.

The reason for this is twofold:
1. Makes the code a little more DRY so that we only have to call a function rather than repeating ``if err != nil``.
2. All errors eventually get thrown as a panic, so they get caught by the ``defer func`` and then print out the "Issue occurred".

Reply:
On it.